codegen/src/main/java/software/amazon/awssdk/codegen/AddShapes.java

```diff
@@ -194,6 +194,7 @@ private MemberModel generateMemberModel(String c2jMemberName, Member c2jMemberDe
         memberModel.setUnionEnumTypeName(namingStrategy.getUnionEnumTypeName(memberModel));
         memberModel.setContextParam(c2jMemberDefinition.getContextParam());
         memberModel.setRequired(isRequiredMember(c2jMemberName, parentShape));
+        memberModel.setSynthetic(shape.isSynthetic());
```

Comment:
+1 Synthetic sounds cool , nice name

Reply:
yep credits to Zoe

Reply:
It's actually Matt who came up with this name :)